test/modules/linear_boltzmann_solvers/transport_steady/reed_balance.lua

```diff
@@ -1,83 +1,79 @@
 -- Standard Reed 1D 1-group problem
 -- Create Mesh
-widths = {2., 1., 2., 1., 2.}
-nrefs = {200, 200, 200, 200, 200}
+widths = { 2., 1., 2., 1., 2. }
+nrefs = { 200, 200, 200, 200, 200 }
 
 Nmat = #widths
 
 nodes = {}
 counter = 1
 nodes[counter] = 0.
 for imat = 1, Nmat do
-  dx = widths[imat] / nrefs[imat]
-  for i = 1, nrefs[imat] do
-    counter = counter + 1
-    nodes[counter] = nodes[counter - 1] + dx
-  end
+    dx = widths[imat] / nrefs[imat]
+    for i = 1, nrefs[imat] do
+        counter = counter + 1
+        nodes[counter] = nodes[counter - 1] + dx
+    end
 end
 
```

Comment:
is the StyLua causing this? Just curious. Because I know I pushed the test case before StyLua (when LuaFormatter was the ephemerous option)

Reply:
My guess would be that it is the CLion plugin @zhardy-lanl is using